javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var numsOnly = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
+      numsOnly.push(values[i]);
+    }
```

Comment:
You don't want to catch all values that are numbers, you want to catch all values that aren't strings. If other data types were included as well,  this would fail.